### DISCUSSION TITLE
Combobox as separate component is amazing idea

### DISCUSSION THREAD

**Discussion by skelawsky** [mantinedev/mantine#5176] [Kind words]
I am rewriting the application from v6 to v7 and as I had big problems before in v6 to get the required functionality, now it was a piece of cake

**Comment by rtivital** [comment]
Thanks for your kind words!

**Comment by juliuslissau** [comment]
I feel like the Combobox is a a shitty change tbh. Just to create a simple `<Select>` where the `label` and `value` for the data options are different, I now need to create a big custom component! Before that used to be 1-2 lines, now it's like 50 lines and a seperate component. What?! And adding search or a custom itemComponent makes it even worse. 

I get it's more flexible, but before was a good medium I would say, this is just super complex compared to what was easy before.

**Reply by skelawsky** [reply]
Don't be afraid to write code! I have the impression it is a headless component now instead of some abstraction as before.
